tools/generate_runtime_dependencies.py

```diff
@@ -0,0 +1,80 @@
+# Python script to determine the runtime dependencies of open source tools hosted on Github
+# The dependencies are stored in a dictionary and printed in a markdown table format
+# Only unique dependencies are stored and printed, to avoid duplication
+# Note: Only run this on z/OS
```

Comment:
Can you add a comment that says what the input and output is and what env vars are required?